src/configSchema.ts

```diff
@@ -85,7 +86,7 @@ function transformDurationString(durationStr: string, ctx: RefinementCtx) {
  * @return path if valid formatting
  */
 function checkValidPathFormat(path: string, ctx: RefinementCtx) {
-	if (sep === "\\" && !path.includes("\\") && !path.includes("/")) {
+	if (!path.includes(sep)) {
```

Comment:
I like the previous better, I'd like to support forward slashes on windows. Not doing so would also be a breaking change

Reply:
The previous was non-functional. That is why I changed it, I think I'd have to use \xA0 explicitly, but I'll do some testing on the escape characters matching in the string and work it out.

Reply:
I got it working, I would also mention we aren't checking validity of the outputDir currently, I think because it could be defaulted to `.` - which begs the question, in the event that it doesn't match the formatting on the line above, do we also accept `.` as the path? 

I tend to think we should require explicit absolute paths...but I'm open to this possibility. I haven't seen it in support ever be used, but that doesn't mean people aren't.

Reply:
Added support for . because if nothing else, it _IS_ valid...i don't think we should explicitly say that it's an option though.